packages/expo-web-browser/src/WebBrowser.types.ts

```diff
@@ -7,6 +7,7 @@ export type OpenBrowserOptions = {
   browserPackage?: string;
   enableBarCollapsing?: boolean;
   showTitle?: boolean;
+  controlsColor?: string;
```

Comment:
```suggestion
  /** iOS only */
  controlsColor?: string;
```
```suggestion
  /*
  * iOS only
  */
  controlsColor?: string;
```

Would this make the comment show up in a helpful popup in VSCode? Do you remember @tsapeta when we researched [this](https://swmansion.slack.com/archives/D0XMLA4RJ/p1560782613060000)?

Reply:
oh yeah, that's right - the first one (`/** iOS only */`) will popup in VSCode

Reply:
Got it, thanks for pointing that out!